googleapiclient/http.py

```diff
@@ -468,7 +469,7 @@ def __init__(self, body, mimetype='application/octet-stream',
     resumable: bool, True if this is a resumable upload. False means upload
       in a single request.
     """
-    fd = StringIO.StringIO(body)
+    fd = BytesIO(body)
```

Comment:
I think `io.BytesIO()` is OK for > Python 2.6.
`six.BytesIO()` is for <= Python 2.5.